cli/tools/init/mod.rs

```diff
@@ -49,7 +49,21 @@ pub async fn init_project(init_flags: InitFlags) -> Result<(), AnyError> {
   let main_test_ts = include_str!("./templates/main_test.ts")
     .replace("{CURRENT_STD_URL}", deno_std::CURRENT_STD_URL_STR);
   create_file(&dir, "main_test.ts", &main_test_ts)?;
-  create_file(&dir, "deno.json", include_str!("./templates/deno.json"))?;
+
+  if init_flags.lib {
+    // Extract the directory name to use as the project name
+    let project_name = dir
+      .file_name()
+      .unwrap_or_else(|| dir.as_os_str())
```

Comment:
This might be problematic - I don't think all directories are valid JSR specifiers, but maybe fine for the first pass and could be improved in the future.